draft-wang-ppm-dap-taskprov.md

```diff
@@ -296,6 +296,27 @@ A protocol participant MUST opt out if the task has expired.
 The behavior of each protocol participant is determined by whether or not they
 opt in to a task.
 
+## Supporting a Global HPKE Configuration {#global-hpke-config}
+
+In DAP, Clients need to know the HPKE configuration of each Aggregator before
+sending reports. (See HPKE Configuration Request in
+{{!DAP=I-D.draft-ietf-ppm-dap-02}}.) However, in a DAP deployment that supports
+`task_prov` extension, if a Client queries an Aggregator's `/hpke_config`
+endpoint with a task ID computed as described in {{construct-task-id}}, the
```

Comment:
It's best to avoid writing text that refers to specific constructions in the base protocol. Suppose for example that the core protocol changed the endpoint to `/get_that_cool_hpke_config`: Then we would have to update this reference in-kind. My suggestion avoids this.
```suggestion
`task_prov` extension, if a Client requests the Aggregator's HPKE configuration
with the task ID computed as described in {{construct-task-id}}, the
```